make/autoconf/jdk-options.m4

```diff
@@ -686,7 +686,7 @@ AC_DEFUN([JDKOPT_ENABLE_DISABLE_COMPATIBLE_CDS_ALIGNMENT],
   UTIL_ARG_ENABLE(NAME: compatible-cds-alignment, DEFAULT: $COMPATIBLE_CDS_ALIGNMENT_DEFAULT,
       RESULT: ENABLE_COMPATIBLE_CDS_ALIGNMENT,
       DESC: [enable use alternative compatible cds core region alignment],
-      DEFAULT_DESC: [disabled],
+      DEFAULT_DESC: [disabled except on linux-aarch64],
```

Comment:
This should not change: for OpenJ9 it's disabled on all platforms.
Line 199 should be deleted (it was removed upstream by accbd5ee60f7775a8c7f9192ca4f967aec8922dd).

Reply:
Restored OpenJ9 change, and also matched upstream deletion.